### PR TITLE
Dịch "index_vn"

### LINKED ISSUE #1790: Dịch "index_vn"
Phần cần dịch: https://github.com/aivivn/d2l-vn/blob/master/chapter_optimization/index_vn.md

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by cuongvng**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #1790

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_optimization/index_vn.md [hunk 1]
```diff
@@ -32,7 +38,9 @@ Nonetheless, the design and analysis of algorithms in the context of convex prob
 It is for that reason that this section includes a primer on convex optimization and the proof for a very simple stochastic gradient descent algorithm on a convex objective function.
 -->
 
-*dịch đoạn phía trên*
+Chương này đề cập sâu hơn đến các thuật toán tối ưu học sâu. Hầu hết tất cả các vấn đề tối ưu xuất hiện trong học sâu là không lồi.
```
Comment (lkhphuc) [inline chapter_optimization/index_vn.md:41]:
```suggestion
Chương này đề cập sâu hơn đến các thuật toán tối ưu học sâu. Hầu hết tất cả các vấn đề tối ưu xuất hiện trong học sâu là không lồi (*nonconvex*). 
```

File: chapter_optimization/index_vn.md [hunk 2]
```diff
@@ -32,7 +38,9 @@ Nonetheless, the design and analysis of algorithms in the context of convex prob
 It is for that reason that this section includes a primer on convex optimization and the proof for a very simple stochastic gradient descent algorithm on a convex objective function.
 -->
 
-*dịch đoạn phía trên*
+Chương này đề cập sâu hơn đến các thuật toán tối ưu học sâu. Hầu hết tất cả các vấn đề tối ưu xuất hiện trong học sâu là không lồi. 
+Tuy nhiên, việc thiết kế và phân tích các thuật toán giải quyết bài toán tối ưu lồi rất hữu ích, điều này đã được kiểm chứng.
```
Comment (lkhphuc) [inline chapter_optimization/index_vn.md:42]:
```suggestion
Tuy nhiên, việc thiết kế và phân tích các thuật toán giải quyết bài toán tối ưu lồi đã được chứng tỏ là rất hữu ích khi áp dụng trong các bài toán không lồi.
```
Mình làm rõ ý convex optim instructive, chứ bản thân nó không hữu dụng trong học sâu.

File: chapter_optimization/index_vn.md [hunk 3]
```diff
@@ -32,7 +38,9 @@ Nonetheless, the design and analysis of algorithms in the context of convex prob
 It is for that reason that this section includes a primer on convex optimization and the proof for a very simple stochastic gradient descent algorithm on a convex objective function.
 -->
 
-*dịch đoạn phía trên*
+Chương này đề cập sâu hơn đến các thuật toán tối ưu học sâu. Hầu hết tất cả các vấn đề tối ưu xuất hiện trong học sâu là không lồi. 
+Tuy nhiên, việc thiết kế và phân tích các thuật toán giải quyết bài toán tối ưu lồi rất hữu ích, điều này đã được kiểm chứng. 
+Do vậy, phần này tập trung vào tối ưu lồi và một thuật toán hạ gradient ngẫu nhiên (stochastic gradient descent) đơn giản áp dụng cho hàm mục tiêu lồi.
```
Comment (lkhphuc) [inline chapter_optimization/index_vn.md:43]:
```suggestion
Do vậy, phần này tập trung vào giới thiệu tối ưu lồi và chứng minh một thuật toán hạ gradient ngẫu nhiên (*stochastic gradient descent*) đơn giản áp dụng cho hàm mục tiêu lồi.
```

File: chapter_optimization/index_vn.md [hunk 4]
```diff
@@ -32,7 +38,9 @@ Nonetheless, the design and analysis of algorithms in the context of convex prob
 It is for that reason that this section includes a primer on convex optimization and the proof for a very simple stochastic gradient descent algorithm on a convex objective function.
 -->
 
-*dịch đoạn phía trên*
+Chương này đề cập sâu hơn đến các thuật toán tối ưu học sâu. Hầu hết tất cả các vấn đề tối ưu xuất hiện trong học sâu là không lồi (*nonconvex*). 
+Tuy nhiên, việc thiết kế và phân tích các thuật toán giải quyết bài toán tối ưu lồi đã được chứng tỏ là rất hữu ích khi áp dụng trong các bài toán không lồi.
```
Comment (lkhphuc) [inline chapter_optimization/index_vn.md:42]:
```suggestion
Tuy nhiên, kiến thức từ việc thiết kế và phân tích các thuật toán giải quyết bài toán tối ưu lồi đã được chứng tỏ là rất hữu ích khi áp dụng trong các bài toán không lồi.
```
cho đủ nghĩa `instructive`.

File: chapter_optimization/index_vn.md [hunk 5]
```diff
@@ -54,4 +62,4 @@ lr-scheduler_vn
 Bản dịch trong trang này được thực hiện bởi:
 
 * Đoàn Võ Duy Thanh
-* 
\ No newline at end of file
+* Nguyễn Văn Cường
```
Comment (cuongvng) [inline chapter_optimization/index_vn.md:65]:
```suggestion
* Nguyễn Văn Cường
* Lê Khắc Hồng Phúc
```

File: chapter_optimization/index_vn.md [hunk 6]
```diff
@@ -12,7 +12,9 @@ Indeed, anyone content with treating optimization as a black box device to minim
 with the knowledge that there exists an array of incantations of such a procedure (with names such as "Adam", "NAG", or "SGD").
 -->
 
-*dịch đoạn phía trên*
+Độc giả theo dõi tuần tự đến chương này của cuốn sách hẳn đã sử dụng nhiều thuật toán tối ưu tiên tiến để huấn luyện các mô hình học sâu.
```
Comment (rootonchair) [inline chapter_optimization/index_vn.md:15]:
```suggestion
Độc giả theo dõi đến chương này của cuốn sách hẳn đã sử dụng nhiều thuật toán tối ưu tiên tiến để huấn luyện các mô hình học sâu. 
```

File: chapter_optimization/index_vn.md [hunk 7]
```diff
@@ -23,7 +25,11 @@ On the other hand, understanding the principles of different optimization algori
 to tune the hyperparameters in a targeted manner to improve the performance of deep learning models.
 -->
 
-*dịch đoạn phía trên*
+Tuy nhiên, để làm tốt, ta cần các kiến thức sâu hơn. 
+Các thuật toán tối ưu rất quan trọng trong học sâu.
```
Comment (rootonchair) [inline chapter_optimization/index_vn.md:29]:
```suggestion
Các giải thuật tối ưu rất quan trọng trong học sâu. 
```

File: chapter_optimization/index_vn.md [hunk 8]
```diff
@@ -32,7 +38,9 @@ Nonetheless, the design and analysis of algorithms in the context of convex prob
 It is for that reason that this section includes a primer on convex optimization and the proof for a very simple stochastic gradient descent algorithm on a convex objective function.
 -->
 
-*dịch đoạn phía trên*
+Chương này đề cập sâu hơn đến các thuật toán tối ưu học sâu. Hầu hết tất cả các vấn đề tối ưu xuất hiện trong học sâu là không lồi (*nonconvex*).
```
Comment (rootonchair) [inline chapter_optimization/index_vn.md:41]:
```suggestion
Chương này đề cập sâu hơn đến các thuật toán tối ưu học sâu. Hầu hết tất cả các vấn đề tối ưu xuất hiện trong học sâu là *không lồi* (*nonconvex*). 
```

File: chapter_optimization/index_vn.md [hunk 9]
```diff
@@ -54,4 +62,5 @@ lr-scheduler_vn
 Bản dịch trong trang này được thực hiện bởi:
 
 * Đoàn Võ Duy Thanh
-* 
\ No newline at end of file
+* Nguyễn Văn Cường
+* Lê Khắc Hồng Phúc
```
Comment (cuongvng) [inline chapter_optimization/index_vn.md:66]:
```suggestion
* Lê Khắc Hồng Phúc
* Phạm Hồng Vinh
```
